src/styles/header.scss

```diff
@@ -0,0 +1,61 @@
+.header {
+  @include spaced-between-centralized;
+
+  width: 100%;
+  padding: 0 50px;
+  box-sizing: border-box;
+
+  &__logo {
+    display: block;
+    width: 40px;
+    height: 40px;
+  }
+
+  &__links {
+    display: flex;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__item {
+    @include spaced-center;
+
+    padding: 0;
+    margin-right: 20px;
+    flex-direction: column;
+  }
+
+  &__item:last-child {
+    margin-right: 0;
+  }
+
+  &__link {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 60px;
+    text-transform: uppercase;
+    text-decoration: none;
+    color: $color-text-primary;
+    transition: 0.5s;
+
+    &:hover {
+      color: $color-button-secondary;
+    }
+  }
+}
+
+.is-active {
+  position: relative;
+  color: $color-button-secondary;
+}
+
+.is-active::after {
```

Comment:
Could be nested into `.is-active` [Reference](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#:~:text=Write%20pseudo%2Dclass%2C%20pseudo%2Delement%20selectors%20inside%20general%20selector.%20As%20well%20as%20media%20queries.)